src/redux/actions/commentActions.js

```diff
@@ -0,0 +1,42 @@
+import api from '../api';
+import actionTypes from './actionTypes';
+
+const getComments = payload => ({
+  type: actionTypes.GET_COMMENT,
+  payload,
+});
+
+const createCommentAction = payload => ({
+  type: actionTypes.CREATE_COMMENT,
+  payload,
+});
+
+const createReplyAction = payload => ({
+  type: actionTypes.CREAT_REPLY,
+  payload,
+});
+
+const deleteCommentAction = payload => ({
+  type: actionTypes.DELETE_COMMENT,
+  payload,
+});
+
+const getAllComments = slug => dispatch => api.get(`/api/articles/${slug}/comments`).then((res) => {
+  dispatch(getComments(res.data));
+});
```

Comment:
Hi, you are not catching any errors in these actions, I think it is necessary you take care of that using `.catch()`